wwwroot/src/actions/view.php

```diff
@@ -55,15 +56,18 @@ function view($path, $action, $title, $content)
         mysqli_query($mysql,"Update `Wiki_Pages` set `Views` = `Views` + 1 where `ID`='$PageID'");
     }
 
-    if($_SESSION['admin'])
+    if(!empty($_SESSION['admin']))
     {
         $content['ExtraNav'] = new Navigation;
         $content['ExtraNav']->Add("Archive This Page", FormatPath("/$path/")."?archive");
         $content['ExtraNav']->Add("Rename This Page", FormatPath("/$path/")."?rename");
     }
 
-    $title[] = FishFormat($PageTitle, "strip");
-    $content['Title'] .= FishFormat($PageTitle);
+    $title[0] = FishFormat($PageTitle, "strip");
+    if(!empty($content['Title'])) {
+        $content['Title'] .= FishFormat($PageTitle);
+    }
+    
```

Comment:
This logic breaks the title. Tested locally, no titles are displayed on the page.

![image](https://github.com/wetfish/wiki/assets/1670549/2a737c65-af79-4c0d-a6c5-d4c8b1553319)